docs-source/usersguide/application/02_running_sims.rst

```diff
@@ -593,6 +593,40 @@ such as :file:`charmm36/water.xml`, which specifies the default CHARMM water mod
 The converted parameter sets come from the `CHARMM36 July 2017 update <http://mackerell.umaryland.edu/charmm_ff.shtml>`_
 and were converted using the `openmm-forcefields <https://github.com/choderalab/openmm-forcefields>`_ package and `parmed <https://github.com/parmed/parmed>`_.
 
+Implicit Solvent
+----------------
+
+The Amber and CHARMM force fields described above can be used with any of the Generalized
+Born implicit solvent models from AMBER.  To use them, include an extra file when
+creating the ForceField.
```

Comment:
Can we give an explicit example of this, such as
```python
# Initialize a force field using the OBC2 GBSA parameters with the AMBER14SB protein force field
forcefield = ForceField('amber14sb.xml', 'implicit/obc2.xml')
```

Reply:
Yes, I'll add that.